polar-core/src/kb.rs

```diff
@@ -26,9 +27,9 @@ impl RuleParamMatch {
 pub struct KnowledgeBase {
     /// A map of bindings: variable name → value. The VM uses a stack internally,
     /// but can translate to and from this type.
-    constants: Bindings,
+    constants: Constants,
     /// Map of class name -> MRO list where the MRO list is a list of class instance IDs
-    mro: HashMap<Symbol, Vec<u64>>,
+    pub mro: HashMap<Symbol, Vec<u64>>,
```

Comment:
for future reference we typically avoid making fields public and instead access through a method. this makes it easier to change the internal representation of data without affecting all callers

Reply:
💯 thanks, that absolutely makes sense.

Reply:
we also definitely haven't followed this convention well so far :)